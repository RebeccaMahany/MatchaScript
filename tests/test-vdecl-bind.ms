void aFunction(int q, float s) {
    print(q);
    print(s);
    print("hello there!");
}


void main() {
    int a = 1;
    char b = 'b';
    string c = "c";
    float d = 1.0;
    int e = a;
    print(e);
    aFunction(e, 2.2);
}
