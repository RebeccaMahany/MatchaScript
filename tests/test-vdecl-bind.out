1
1
2.200000
hello there!
